examples/util/openvino/OpenVINOModelImporter.js

```diff
@@ -681,6 +737,90 @@ class OpenVINOModelImporter {
 
           opCode = this._nn.SOFTMAX;
         } break;
+        case 'Normalize': {
+          const input = node.inputs[0];
+          const inDims = input.shape();
+          const inputId = this._getTensorId(input);
+          console.log(`  input shape: [${input.shape()}]`);
+
+          const acrossSpatial = node.getInt('across_spatial');
+          const channelShared = node.getInt('channel_shared');
+          const eps = node.getFloat('eps');
```

Comment:
do we support `eps`?

Reply:
No, the NNAPI not support, so we do not.

Reply:
Then I suggest to remove this line and add a comment that says we don't support `eps` now.

Reply:
I delete Normalization supportting.